Packs/CloudIncidentResponse/Playbooks/playbook-Cloud_Data_Exfiltration_Response.yml

```diff
@@ -0,0 +1,1179 @@
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 22d468a7-b5b0-47bd-8376-e59f74fab8e1
+    type: start
+    task:
+      id: 22d468a7-b5b0-47bd-8376-e59f74fab8e1
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "73"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 680,
+          "y": -120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 676a1dee-45cb-4f60-8e6a-125fa23d9e42
+    type: title
+    task:
+      id: 676a1dee-45cb-4f60-8e6a-125fa23d9e42
+      version: -1
+      name: Entity Enrichment
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "59"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1080,
+          "y": 320
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: 3c7469d0-c6e8-4523-8440-2b90681d38a2
+    type: title
+    task:
+      id: 3c7469d0-c6e8-4523-8440-2b90681d38a2
+      version: -1
+      name: Enumeration Alert Search
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "13"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 280,
+          "y": 320
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 1bbbc703-b46d-4532-83c6-e51b27cf570d
+    type: regular
+    task:
+      id: 1bbbc703-b46d-4532-83c6-e51b27cf570d
+      version: -1
+      name: Search alerts for cloud enumeration activity
+      description: |-
+        Searches Cortex XSIAM alerts. A summarized version of this scrips is available with the summarizedversion argument.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.12/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: SearchIncidentsV2
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "78"
+    scriptarguments:
+      details:
+        simple: '*enumeration*'
```

Comment:
add another filter to correlate between the enumeration alerts and the incident specifics.